CHANGELOG.md

```diff
@@ -70,7 +70,7 @@ New Features
 - `SED`s can now be constructed with flexible units via the `astropy.units`
   module. (#789).
 - Added new surface brightness profile, 'InclinedExponential'. This represents
-  the 2D projection of the 3D profile:
+  the 2D projection the 3D profile:
```

Comment:
This actually arose because I noticed that "of" was missing, and accidentally fixed it on the master branch instead of this one. I thought I'd reverted that and made the change here, but it appears that didn't take for some reason.